test/unit/node/generated-code-dependencies.js

```diff
@@ -347,17 +347,26 @@ suite('context test', function() {
     });
   });
 
+  function checkFile(outDir, goldenDir, filename) {
+    var actualContents = fs.readFileSync(path.resolve(outDir, filename));
+    var actual = actualContents.toString().split('\n');
```

Comment:
For these I think splitting is going to lead to worse error message. The standard one shows the diff and by splitting we miss cross line diffs.